documentation/modules/ROOT/pages/operations/kubernetes.adoc

```diff
@@ -0,0 +1,369 @@
+[id="deploying-debezium-on-Kubernetes"]
+= Deploying {prodname} on Kubernetes
+
+:linkattrs:
+:icons: font
+:toc:
+:toclevels: 3
+:toc-placement: macro
+
+toc::[]
+
+{prodname} can be easily deployed on https://kubernetes.io/[Kubernetes], an open-source container management platform.
+The deployment leverages the https://strimzi.io/[Strimzi] project, which aims to simplify the deployment of Apache Kafka on Kubernetes by means of _custom resources_.
+
+[NOTE]
+====
+For testing your deployment, you can use https://minikube.sigs.k8s.io/docs/[minikube], which starts a Kubernetes cluster on your local machine.
+See the https://minikube.sigs.k8s.io/docs/start/[minikube documentation] for details on how to install minikube on your machine.
+If you want to test a {prodname} deployment as described in this doc completely on minikube, you need to set up an unsecure container image registry on minikube.
+To do so, you need start minikube with the `--insecure-registry` flag:
+
+[source,subs="attributes",options="nowrap"]
+----
+$ minikube start --insecure-registry "10.0.0.0/24"
+----
+
+`10.0.0.1` is the default service cluster IP, so this setup allows pulling images inside the whole cluster.
+See https://minikube.sigs.k8s.io/docs/handbook/registry/#enabling-insecure-registries[minikube documentation] for more details.
+You also need to enable `registry` minikube addon:
+
+[source,subs="attributes",options="nowrap"]
+----
+$ minikube addons enable registry
+----
+
+====
+
+== Prerequisites
+
+To keep containers separated from other workloads on the cluster, create a dedicated namespace for {prodname}.
+In the remainder of this document, the `debezium-example` namespace will be used:
+
+[source,subs="attributes",options="nowrap"]
+----
+$ kubectl create ns debezium-example
+----
+
+As mentioned above, for the {prodname} deployment we will use Strimzi, which manages the Kafka deployment on Kubernetes.
+Please see the https://strimzi.io/docs/operators/latest/deploying.html[Strimzi deployment documentation] for more details on how to deploy Strimzi on your Kubernetes cluster.
+
+The most simple way is to install Strimzi operator.
+Operators require Operator Lifecycle Manager to be install on your cluster.
+If you don't have it installed, you can install it by:
```

Comment:
So Strimzi really _is_ an operator, there's no way to use Strimzi without its operator. I'll reword this slightly:

> This simplest way for installing Strimzi is using the Operator Lifecycle Manager ...

That said, it's a nice improvement!

```suggestion
The simplest way for installing Strimzi is through the https://olm.operatorframework.io/[Operator Lifecycle Manager] (OLM).
If you don't have OLM installed on your cluster yet, you can install it by running the following command:
```